internal/deploy/bootstrap.go

```diff
@@ -0,0 +1,77 @@
+package deploy
+
+import (
+	"bytes"
+	"fmt"
+	"regexp"
+
+	"github.com/kyma-project/cli/internal/kube"
+	"github.com/kyma-project/cli/internal/kustomize"
+)
+
+// Bootstrap deploys the kustomization files for the prerequisites for Kyma.
+// Returns true if the Kyma CRD was deployed.
+func Bootstrap(kustomizations []string, k8s kube.KymaKube, dryRun bool) (bool, error) {
+	defs := []kustomize.Definition{}
+	// defaults
+	if len(kustomizations) == 0 {
+		lm, err := kustomize.ParseKustomization("https://github.com/kyma-project/lifecycle-manager/operator/config/default")
+		if err != nil {
+			return false, err
+		}
+		mm, err := kustomize.ParseKustomization("https://github.com/kyma-project/module-manager/operator/config/default")
```

Comment:
Could you please move the values of the kustomization paths for lifecycle-manager and module-manager into constants, for easier configuration if they need to be changed later on.

Reply:
That is a god point yes, will move it.